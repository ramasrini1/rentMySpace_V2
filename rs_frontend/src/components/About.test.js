import React from "react";
import { render } from "@testing-library/react";
import About from "./About";
import { MemoryRouter } from "react-router";

it("matches snapshot with logo", function () {
  const { asFragment } = render(
      <MemoryRouter>
        <About
        />
      </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});

