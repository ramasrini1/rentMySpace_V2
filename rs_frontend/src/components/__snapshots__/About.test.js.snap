// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot with logo 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h4
      class="mt-4"
    >
      Rent My Space
    </h4>
    <div
      class="mt-4"
    >
      Have you gone to a beach on weekends and struggled to get a parking spot or had trouble in finding a place to park near big events like the 4th of July firework ? 
    </div>
    <div
      class="mt-2"
    >
      Or do you wish to host a swimming pool party but are restriced because you don't have swimming pool in your house?
    </div>
    <div
      class="mt-2"
    >
      <b>
        ‘Rent My Space’
      </b>
       app can provide solutions to these problems. This app provides a listing of all the properties for renters to book. The renters can book the property for using the driveway, backyard swimming pool or other spaces.
    </div>
    <div
      class="mt-2"
    >
      If you don't have an account 
      <a
        href="/signup"
      >
        Sign Up
      </a>
       now and host a property or book one depending on your needs. Make your next event hosting or trips to beaches a hassle free process.
    </div>
  </div>
</DocumentFragment>
`;
